topics/transcriptomics/tutorials/ref-based/tutorial.md

```diff
@@ -1294,10 +1296,10 @@ Now we would like to extract the most differentially expressed genes due to the
 >
 >    > ### {% icon comment %} Comment
 >    >
->    > The file with the independent filtered results can be used for further downstream analysis as it excludes genes with only few read counts as these genes will not be considered as significantly differentially expressed.
+>    > The file with the independently filtered results can be used for further downstream analysis as it excludes genes with only a few read counts, as these genes will not be considered as significantly differentially expressed.
 >    {: .comment}
 >
->    We will now select only the genes with an absolute fold change (FC) > 2
+>    We will now select only the genes with an absolute fold change (FC) > 1.
```

Comment:
this seems to contradict the input we actually use for the tool, `abs(c3)>1`? I assume I'm missing something here, but maybe this can be explained in a bit more detail.

Reply:
I somehow missed the fact that we are dealing with a logarithm, thanks to @EngyNasr for pointing that out. I added a sentence to clarify.